validphys2/src/validphys/config.py

```diff
@@ -6,6 +6,8 @@
 """
 import logging
 import pathlib
+import numpy as np
+import pandas as pd
```

Comment:
They should be below collections to follow the "standard-lib, third-party, first-pary" rule.

Reply:
And add a blank line to separate the two groups.